src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.html

```diff
@@ -1,8 +1,13 @@
 <ul class="edit-page-nav">
```

Comment:
QA stuff: if you place the mouse in the space between the text and the icon the tool start flickering.

Reply:
nope it does not happen

Reply:
![screen capture on 2018-04-06 at 12-03-01](https://user-images.githubusercontent.com/751424/38436705-96a582ae-3992-11e8-87da-6939d1ec8edf.gif)

Reply:
Hi, please test again with the proper key message key from `core`, it seems on your demo that since the text is not the right one, then it causes that issue
https://monosnap.com/file/GL3B4lqJvF4tvbM33bIw8VwIZcU5iC